app/Factories/LinkFactory.php

```diff
@@ -63,13 +63,6 @@ public static function createLink($long_url, $is_secret=false, $custom_ending=nu
         }
 
         if (isset($custom_ending) && $custom_ending !== '') {
-            // has custom ending
```

Comment:
I have removed this since it should already be checked by this point, however if we want to keep it here we should use a validator.

Reply:
Do you mean that the ending is already checked through the "link availability" endpoint on the frontend? We always should validate user input on the server side as well, because there is nothing stopping a user from creating a link even if the ending is invalid.

Reply:
It's already validated when creating a link using the front end, upon double checking it seems to be missing from the API controller validation, so I have added it there as well.